onnxruntime/core/graph/graph.cc

```diff
@@ -549,6 +549,15 @@ const Graph* Node::GetGraphAttribute(const std::string& attr_name) const {
   return const_cast<Node*>(this)->GetMutableGraphAttribute(attr_name);
 }
 
+std::vector<gsl::not_null<const Graph*>> Node::GetSubgraphs() const {
+  std::vector<gsl::not_null<const Graph*>> subgraphs;
+  subgraphs.reserve(attr_to_subgraph_map_.size());
+  std::transform(attr_to_subgraph_map_.cbegin(), attr_to_subgraph_map_.cend(), std::back_inserter(subgraphs),
```

Comment:
I hope this back_inserter does not interfere with the fact that you do a reserve right before (as in it will not try to insert at the end of the `attr_to_subgraph_map_.size()` elements of the subgraphs vector).

Reply:
reserve != resize, so it would work as expected.